irohad/ametsuchi/impl/postgres_query_executor.hpp

```diff
@@ -95,23 +99,34 @@ namespace iroha {
        * predicate pred
        */
       template <typename RangeGen, typename Pred>
-      auto getTransactionsFromBlock(uint64_t block_id,
-                                    RangeGen &&range_gen,
-                                    Pred &&pred);
+      std::vector<std::unique_ptr<shared_model::interface::Transaction>>
+      getTransactionsFromBlock(uint64_t block_id,
+                               RangeGen &&range_gen,
+                               Pred &&pred);
 
       /**
-       * Execute query in F and return builder result from B
+       * Execute query in F and return query response from B
        * Q is query tuple, P is permission tuple
```

Comment:
Maybe name Q a QueryTuple instead? Same for the PermissionTuple

Reply:
I am always for giving template parameters meaningful names :)